podpac/core/utils.py

```diff
@@ -475,3 +475,70 @@ def resolve_bbox_order(bbox, crs, size):
     size = size[::r]
 
     return {"lat": [lat_start, lat_stop, size[0]], "lon": [lon_start, lon_stop, size[1]]}
+
+
+def probe_node(node, lat=None, lon=None, time=None, alt=None, nested=False):
+    """"""
+
+    def partial_definition(key, definition):
+        new_def = OrderedDict()
+        for k in definition:
+            new_def[k] = definition[k]
+            if k == key:
+                return new_def
+
+    def flatten_list(l):
+        nl = []
+        for ll in l:
+            if isinstance(ll, list):
+                lll = flatten_list(ll)
+                nl.extend(lll)
+            else:
+                nl.append(ll)
+        return nl
+
+    c = [(v, d) for v, d in zip([lat, lon, time, alt], ["lat", "lon", "time", "alt"]) if v is not None]
+    coords = podpac.Coordinates([[v[0]] for v in c], [[d[1]] for d in c])
+    with podpac.settings:
```

Comment:
I'm abusing the cache here to make my life easy for figuring out the actual "active" state below. If anyone can think of a better way to do this, I'm all ears.

Reply:
@jmilloy pointed out a much better way to this.